aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -1448,7 +1448,7 @@ def build_mission(self, num_nodes, aviary_inputs):
         for variable in variables_to_scale:
             self.add_scaling_exec_comp(scaling_group, variable, num_nodes=num_nodes)
         self.add_scaling_exec_comp(scaling_group, 'fuel_flow_rate', num_nodes=num_nodes,
-                                   alias=Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE)
+                                   alias=Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE, sign_scalar=-1.0)
```

Comment:
does this mean we should be taking the "Negative" out of the variable description